### DISCUSSION TITLE
New relude maintainers!

### DISCUSSION THREAD

**Discussion by andywhite37** [reazen/relude#311] [Announcements]
Hi all! As many of you probably noticed, the libraries in the `reazen` ecosystem have gone a bit stale, and have felt a bit abandoned. @mlms13 and I were once production users and maintainers of the `relude` libraries, but we are no longer actively working in what used to be the `BuckleScript/ReasonML` community, so we lost the energy and motivation to continue working on `relude` at the rate we once were.

We've been collaborating on these libraries for a long time with @johnhaley81 and I'm happy to announce today that John has been added to the `reazen` org as a new Owner and co-maintainer of the `relude` libraries. He will likely be bringing in some more people from his company Qwick too. It will take some time to incorporate them into the mix, but I imagine there will be some activity on the libraries again soon.

This basically means that there will once again be some love for `relude` from actual production users. I can't make any guarantees where the library might go in the future, but we can figure out the path forward as we go. Some of the upcoming decisions might involve things related to `rescript`, `melange`, and/or vanilla `ocaml` support.

@mlms13 and I will still be around, but I probably won't be working on the project very actively going forward. I will let @mlms13 speak on his own behalf about anything I said above. I'm happy to have @johnhaley81 and the folks from Qwick to keep the library moving forward!

**Comment by flash-gordon** [comment]
@andywhite37 @mlms13 thanks for your work! ❤️

**Comment by johnhaley81** [comment]
Hi all!

We are active and long-time users of this library in production and are excited to take a bigger role in growing `relude`. Like @andywhite37 said, I'll be adding some more engineers from Qwick on as maintainers but we're also very interested in how other people are using the library as well. 

And again, thank you @andywhite37 and @mlms13 for everything you've done!

**Reply by jakubrpawlowski** [reply]
👋 Hello and thank you as well! Can't imagine coding without relude - looking forward to many years to come!

**Comment by maxkorp** [comment]
I'm excited to help out! I've been using Relude since 2019 and I can't imagine using Reason without it. Thank you so much you two for everything you've done.
